src/main/native/pairhmm/Context.h

```diff
@@ -73,7 +73,7 @@ class ContextBase
         small = t;
       }
 
-      if (std::isinf(small) == -1 || std::isinf(big) == -1)
+      if (std::isinf(small) || std::isinf(big))
```

Comment:
Open on real intent of this code. Need to better understand if this is proper to do.

Reply:
small and big are sorted by size at this point.  If either is infinity return big.  Sounds fine to me.